### DISCUSSION TITLE
Missing GVM_DATA in GSA

### DISCUSSION THREAD

**Discussion by devMarc5598** [Secure-Compliance-Solutions-LLC/GVM-Docker#273] [Q&A]
Hi everyone,
I got a Problem with my GVM DATA.
In /var/lib/gvm/data-objects/gvmd is all the data for 20.08, 21.04 and 21.10 and the Version is 202108091427.
But on the Web Interface is still no data… The compliance guidelines, port lists, report formats and scan configurations are empty.
It seems like the GSA don’t use this data in var/lib/gvm/data-objects/gvmd.

I did all the commands to get the data but it will not appear in the GSA.
greenbone-feed-sync --type GVMD_DATA
greenbone-feed-sync --type SCAP
greenbone-feed-sync --type CERT

and i tried it with the rsync command too
rsync -avrP rsync://feed.community.greenbone.net:/data-objects/gvmd/ .
reboot
# GVM versions

gsad: 21.4.2
gvmd: 21.4.3
openvas-scanner: 21.4.2
gvm-libs: 21.4.2
# Environment
Operating system: Alpine Linux v3.14
Kernel: 5.3.18-lp152.75-default

I hope someone can help me with this issue.
Thanks!

**Comment by Dexus** [comment]
What tells you the status page of the feeds in the webUi?

Von meinem iPhone gesendet

**Reply by devMarc5598** [reply]
The feed status is uptodate.
![image](https://user-images.githubusercontent.com/72303024/132858290-ba6e8464-3399-45cb-bc14-159a214544cd.png)
But there are still no data in the portlists etc.. I think gvmd dont picks the synced data up and dont loads them in the postgres db. 
Because normally after gvmd has loaded them they are available in the web UI.

**Comment by Dexus** [comment]
Please remove your volumes! And restart the deployment your deployment is broken you restarted it maybe to early…

Von meinem iPhone gesendet

**Reply by devMarc5598** [reply]
Is it possible that I just pulled the wrong Image?
I pulled with the command: docker pull securecompliance/gvm:latest
So i pulled this one: (with just 215mb)
https://hub.docker.com/layers/securecompliance/gvm/latest/images/sha256-b11df7b3ec5416d6409920ea2400d7ad65a193ae3bd077a13b2dbc1dead4939a?context=explore

Should i pull a image with a specific tag like this one instead?
https://hub.docker.com/layers/securecompliance/gvm/21.4.3-v1-data-full/images/sha256-ae8ac631f48ebb825c1135d513d3bd059f0b6ba4d2a346b587d3da9f1b888e7e?context=explore

**Reply by Dexus** [reply]
I think it has to do, that we had in our docs the wrong path for the database? And you use them?
This can be also possible, normally this should not make any difference if you use the volumes, if you use it only without any volumes - this change all then the right one should be `-data-full`

**Reply by devMarc5598** [reply]
> I think it has to do, that we had in our docs the wrong path for the database? And you use them?
> This can be also possible, normally this should not make any difference if you use the volumes, if you use it only without any volumes - this change all then the right one should be `-data-full`

So what do you mean? Should I stop the container, remove the current volumes and restart it or should I pull a other image and run it with this one?
My volumes in the docker-compose file are:
- /openvas_data_neu/postgres-db:/opt/database
- /openvas_data_neu/openvas-plugins:/var/lib/openvas/plugins
- /openvas_data_neu/gvm:/var/lib/gvm
- /openvas_data_neu/ssh:/etc/ssh
From https://securecompliance.gitbook.io/projects/docker_compose

And how do I know which image with which tag should be used? Just: docker pull securecompliance/gvm (without any tag)?
